packages/nodes-base/nodes/Gitlab/Gitlab.node.ts

```diff
@@ -1319,22 +1319,22 @@ export class Gitlab implements INodeType {
 		const items = this.getInputData();
 		const returnData: INodeExecutionData[] = [];
 
-		let _credentials;
```

Comment:
since `_credentials` used anywhere, is this dead-code copy-pasted from elsewhere?